### PR TITLE
POT documentation updates

### PR DESCRIPTION
**Pull request by tsavina**
http://openvino-docs.inn.intel.com/tsavina-pot-1/pot_README.html

### CODE REVIEW

File: tools/pot/docs/InstallationGuide.md [hunk 1]
```diff
@@ -18,50 +18,3 @@ The simplest way to get the Post-training Optimization Tool and OpenVINO&trade;
 3. To install POT and other OpenVINO&trade; developer tools, run `pip install openvino-dev`.
 
 Now the Post-training Optimization Tool is available in the command line by the `pot` alias. To verify it, run `pot -h`.
-
```
Comment (AlexKoff88) [inline tools/pot/docs/InstallationGuide.md:21]: This document contains a phrase like "There are two ways how to install the POT on your system:" followed by the list of options. But essentially now there is only one option. I suggest removing or rephrasing this.

Comment (tsavina) [reply tools/pot/docs/InstallationGuide.md:21]: Thanks! This was a point I wanted to discuss, do we want to include the custom package builds option or mention it anywhere in this doc?

**Comment by AlexKoff88** [comment]
@tsavina, can you please resolve conflicts? This should also trigger a new CI run to confirm that everything is ok with this PR.

**Comment by tsavina** [comment]
done

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
A back mege to release branch is required as well.

**Pull request merged**